icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/format/MeasureUnitGeneratorTest.java

```diff
@@ -293,22 +303,39 @@ public int compareTo(OrderedPair<F, S> other) {
         CLDR_NAME_REMAP.put("pound-force-foot",            "pound-foot");
     }
 
+    private static final String ICU_ROOT = findIcuRoot();
+
+    private static String findIcuRoot() {
+        URL x = MeasureUnitGeneratorTest.class.getResource(".");
+        String classFile = x.getFile();
+        int idx = classFile.indexOf("/icu4j/main/common_tests/target/");
+        if (idx != -1) {
+            return classFile.substring(0, idx);
+        } else {
+            return "${icuroot}";
+        }
+    }
+
     @Test
-    public void testZZZ() {
+    public void generateUnitTestsUpdate() throws IOException {
         // various generateXXX calls go here, see
         // docs/processes/release/tasks/updating-measure-unit.md
         // use this test to run each of the following in succession
-        generateConstants("76"); // for MeasureUnit.java, update generated MeasureUnit constants
-        generateBackwardCompatibilityTest("76"); // for MeasureUnitTest.java, create TestCompatible74
-        generateCXXHConstants("76"); // for measunit.h, update generated createXXX methods
-        generateCXXConstants(); // for measunit.cpp, update generated code
-        generateCXXBackwardCompatibilityTest("74"); // for measfmttest.cpp, create TestCompatible74
-        updateJAVAVersions("74"); // for MeasureUnitTest.java, JAVA_VERSIONS
+        if (System.getProperty("generateMeasureUnitUpdate") != null) {
```

Comment:
totally optional: instead of a command-line option, could we mark the test as "Ignore" so that it's normally not run, but then run it explicitly?

Reply:
With `@Ignore` you must uncomment it, run it, then comment it again.
It think it is clunkier than command line?
(but I defer to people that actually did this, they know best what it feels like :-)

Reply:
I was assuming that if you explicitly ask for an Ignore test to be run, it ignores Ignore. I haven't tried it. If the command line option is best, then let's use it.

Reply:
I tried adding `@Ignore`.
It is always ignored, explicitly running it does not work, the method is ignored.
Tried from both command line and Eclipse.